### PR TITLE
Implement toggle fullscreen feature (Close #625) by @Chromecube

### LINKED ISSUE #621: Option to start windowed
**Perceived idea awesomeness**
- [ ] Wow, you don't have this yet???
- [ ] Would be super cool.
- [x] I'm not sure a lot of people would use it, but I'd love to have it.
- [ ] Kind of a gadget, honestly, but why not?

**Perceived idea difficulty**
- [ ] Quit your job first.
- [ ] Would need some work, but nothing impossible.
- [x] Shouldn't be too hard to make.
- [ ] Like, *I* could do it.

---

What it says on the tin. I work nearly exclusively windowed, so an option to start windowed would make my life easier. If not for the fact that I don't know how to develop on Electron itself, I could probably do it since I know JS ;)

### PR DESCRIPTION
**Pull request by Chromecube**
Implementing changes requested in #621 🎉 

Close #621

## Changes

- When pressing `F11`, the function `window.toggleFullScreen()` is called now.
- This function is defined in `_renderer.js`, it does the same thing as before as well as saving the new configuration value `startWindowed`.
- I tested it on Windows.

## Possible bugs

I'm not quite sure whether this is relevant, but before this commit the fullscreen mode was toggled using `electron.remote.BrowserWindow.getFocusedWindow()`, I used `electron.remote.BrowserWindow.getCurrentWindow()` instead as this is used more frequently throughout the project. If this leads to any problems it probably should be changed to `getFocusedWindow()` again.

## Making changes

If my way to implement this feature can not keep up with quality standards, please feel free to make (or request) changes! 

_Edit: This PR seems to be in conflict with the master branch because I added a new configuration value._

### CODE REVIEW

**Comment by GitSquared** [comment]
I fixed the merge conflicts - you were missing a commit I pushed last night on `master`.

This is a great PR, however like I said in my [comment](https://github.com/GitSquared/edex-ui/issues/621#issuecomment-549547750) on #621 it would be great if we could avoid adding another settings option.

Instead of storing the fullscreen state in the settings as `startWindowed`, could you maybe write a `.lastState` file in eDEX's config dir?

**Comment by Chromecube** [comment]
Switched to a file called `lastWindowState.json`

File: src/_boot.js [hunk 1]
```diff
@@ -106,6 +107,13 @@ if (!fs.existsSync(shortcutsFile)) {
     ], "", 4));
     signale.info(`Default keymap written to ${shortcutsFile}`);
 }
+//Create default shortcuts file
```
Comment (GitSquared) [inline src/_boot.js]: Should be: `Create default windowState file`

File: src/_renderer.js [hunk 2]
```diff
@@ -789,7 +791,6 @@ window.writeSettingsFile = () => {
         nocursor: (document.getElementById("settingsEditor-nocursor").value === "true"),
         iface: document.getElementById("settingsEditor-iface").value,
         allowWindowed: (document.getElementById("settingsEditor-allowWindowed").value === "true"),
-        keepGeometry: (document.getElementById("settingsEditor-keepGeometry").value === "true"),
```
Comment (GitSquared) [inline src/_renderer.js]: Please don't delete this!

**Comment by Chromecube** [comment]
Sorry about that, I was a bit in a hurry 😅

**Pull request merged**